From 8ceba39fb3c34d362e9fc5f93230bc687b654167 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Thu, 23 Jun 2011 12:42:30 -0300
Subject: [RHEL6 qemu-kvm PATCH 114/115] usb-bus: Add knowledge of USB_SPEED_SUPER to usb_speed helper

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1308832951-8995-115-git-send-email-kraxel@redhat.com>
Patchwork-id: 27922
O-Subject: [RHEL-6.2 kvm PATCH 114/115] usb-bus: Add knowledge of USB_SPEED_SUPER to usb_speed helper
Bugzilla: 561414 632299 645351 711354
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Hans de Goede <hdegoede@redhat.com>

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
(cherry picked from commit 290d26d2e7510fbbcb68a765695b5838af60b3f0)
---
 hw/usb-bus.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/usb-bus.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/hw/usb-bus.c b/hw/usb-bus.c
index e596df8..81bb49e 100644
--- a/hw/usb-bus.c
+++ b/hw/usb-bus.c
@@ -270,6 +270,7 @@ static const char *usb_speed(unsigned int speed)
         [ USB_SPEED_LOW  ] = "1.5",
         [ USB_SPEED_FULL ] = "12",
         [ USB_SPEED_HIGH ] = "480",
+        [ USB_SPEED_SUPER ] = "5000",
     };
     if (speed >= ARRAY_SIZE(txt))
         return "?";
-- 
1.7.3.2

